### DISCUSSION TITLE
Formula request: touchHLE

### DISCUSSION THREAD

**Discussion by upintheairsheep** [Homebrew/discussions#4718] [Writing Formulae/Casks]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

Hello, can you add a formula for touchHLE?
https://github.com/hikari-no-yume/touchHLE/releases/
The macOS package is a zip file with a portable cli binary, "touchHLE" inside, and a userdata folder, touchHLE_sandbox is in the folder when ran with an app that saves data, and a text file with settings, touchHLE_options.txt inside. Running it with a compatible app opens GUI window. I only have knowledge on how to create cask files at this point.

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew
